libopae/plugins/xfpga/sysfs.c

```diff
@@ -1173,22 +1173,66 @@ fpga_result get_port_sysfs(fpga_handle handle, char *sysfs_port)
 	return FPGA_OK;
 }
 
-// get fpga device id
-fpga_result get_fpga_deviceid(fpga_handle handle, uint64_t *deviceid)
+enum fpga_hw_type opae_id_to_hw_type(uint16_t vendor_id, uint16_t device_id)
+{
+	enum fpga_hw_type hw_type = FPGA_HW_UNKNOWN;
+
+	if (vendor_id == 0x8086) {
+
+		switch (device_id) {
+		case 0xbcbc:
```

Comment:
I'd prefer to keep it localized here..  the idea is to prevent their direct usage in the code, because a single device ID doesn't capture both the PF/VF aspect.  eg, there was a check for 0xbcc0, but what about 0xbcc1?  So long as we use fpga_hw_type, we're ok for both.